<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ExportAllViewsDialogBase</class>
 <widget class="QWidget" name="ExportAllViewsDialogBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>368</width>
    <height>246</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Export All Views</string>
  </property>
  <property name="sizeGripEnabled" stdset="0">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QVBoxLayout">
     <item>
      <widget class="QLabel" name="kURLRequesterLabel">
       <property name="toolTip">
        <string>The base directory used to save the images</string>
       </property>
       <property name="text">
        <string>&amp;Directory to save the diagrams in:</string>
       </property>
       <property name="wordWrap">
        <bool>false</bool>
       </property>
       <property name="buddy">
        <cstring>m_kURL</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="KUrlRequester" name="m_kURL">
       <property name="toolTip">
        <string>The base directory used to save the images</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="ui_imageTypeLayout"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="ui_imageResolutionLayout"/>
   </item>
   <item>
    <widget class="QCheckBox" name="m_useFolders">
     <property name="toolTip">
      <string>Create in the target directory the same tree structure used
in the document to store the views</string>
     </property>
     <property name="whatsThis">
      <string>The views are stored in folders in the document. The same tree structure used in the document to store the views can be created in the selected base directory with this option.
Only the folders made by the user are created in the base directory (Logical view, use case view and so on are not created).</string>
     </property>
     <property name="text">
      <string>Use &amp;folders</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KUrlRequester</class>
   <extends>QFrame</extends>
   <header>KUrlRequester</header>
  </customwidget>
 </customwidgets>
 <includes>
  <include location="global">KUrlRequester</include>
 </includes>
 <resources/>
 <connections/>
</ui>
